lib/diagnostic.js

```diff
@@ -6,34 +6,54 @@
 // splits it into an array of normalized words
 // (uppercase strings without punctuation)
 // and returns that array.
-const normalizeWords = function (/* */) {
-  /* your response here */
+const getNormalizedWords = function getNormalizedWords (text) {
+  let words = text.split(/\s+/);
+  let normalizedWords = [];
+  for (let i = 0; i < words.length; i++) {
+    let word = words[i].replace(/\w+/, '');
```

Comment:
The regular expression ```\w+``` should be ```\W+```.  A lowercase w means match all word characters, whereas a capital W means match all non-word characters.